### ISSUE TITLE
Build failure: floorp-unwrapped

### ISSUE THREAD

**Issue by h7lv2** [NixOS/nixpkgs#321244]
### Steps To Reproduce

1. build `floorp-unwrapped`

### Build log

https://gist.github.com/h7lv2/535b4287714794abc00eb458defff0ed

### Additional context

As a dependency for `floorp`, this also causes `floorp` to fail building

### Notify maintainers

@christoph-heiss 

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.9.3-cachyos, NixOS, 24.11 (Vicuña), 24.11.20240612.57d6973`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - nixpkgs: `/nix/store/dydg48djlykksz8cxq0xjplyxpa9pvf4-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by ghpzin** [comment]
Should have been fixed with https://github.com/NixOS/nixpkgs/pull/320114
Built fine on hydra:
https://hydra.nixos.org/build/263542576
And fix should be in `nixos-unstable`:
https://nixpk.gs/pr-tracker.html?pr=320114

**Comment by christoph-heiss** [comment]
> Should have been fixed with #320114 Built fine on hydra: https://hydra.nixos.org/build/263542576 And fix should be in `nixos-unstable`: https://nixpk.gs/pr-tracker.html?pr=320114

Yes, exactly, this has already been fixed with #320114 a few days ago.
